packages/venia-ui/lib/components/ProductListing/product.css

```
@@ -0,0 +1,71 @@
+.root {
+    display: grid;
+    grid-gap: 0.5rem 1rem;
+    grid-template-areas:
+        'image name quantity kebab'
+        'image options quantity kebab'
+        'image price quantity kebab';
+    grid-template-columns: 100px 2fr max-content min-content;
```

```suggestion
    grid-template-columns: 100px 1fr max-content min-content;
```

---
If you only have one flexible column, `1fr` is enough.

---

Fixed in 0ac27f8